### ISSUE TITLE
Tree items not announced as selected

### ISSUE THREAD

**Issue by paulGeoghegan** [shoelace-style/webawesome-alpha#12]
Non-selected tree items are announced as not selected. However, the currently selected item is not announced as selected with NVDA.

**Comment by KonnorRogers** [comment]
@paulGeoghegan Do you mind mentioning the browser?

I found in Chrome / Safari it reads the same as the APG example found here:

https://www.w3.org/WAI/ARIA/apg/patterns/treeview/examples/treeview-1a/

but in Firefox (127.0.1), it does not read "selected" both in Web Awesome trees, and in the APG example linked above.

**Comment by paulGeoghegan** [comment]
@KonnorRogers FF again and totally on me. It would be great to get some issue templates set up to stop people like me from not including very important details LOL.

**Comment by KonnorRogers** [comment]
@paulGeoghegan Good Idea! I copied the one from Shoelace!

https://github.com/shoelace-style/webawesome-alpha/pull/15
